var/spack/repos/builtin/packages/mumps/package.py

```diff
@@ -262,3 +262,11 @@ def install(self, spec, prefix):
                 os.system('./dsimpletest < input_simpletest_real')
                 if '+complex' in spec:
                     os.system('./zsimpletest < input_simpletest_cmplx')
+
+    @property
+    def libs(self):
```

Comment:
Can u please split this PR in small independent ones? Openblas, openmpi, hypre, etc — those changes are independent